locales/notes.ca.properties

```diff
@@ -0,0 +1,80 @@
+# General note/notebook operations
+notebooks=Blocs de Notes
+tap-anywhere-to-start=Premeu a qualsevol lloc per iniciar
+note-info=Informacio de la nota
+add-to-contacts=Afegir a Contactes
+new-notebook=Crear Bloc de Notes
+notebook-all=Totes les notes
+notebook-trash=Paperera
+notebook-action-title=Editar Bloc de notes
+notebook-action-rename=Canviar el nom
+notebook-action-delete=Eliminar
+prompt-rename-notebook=Canviar el nom del Bloc de notes:
+prompt-delete-notebook=Premeu OK per eliminar aquest Bloc de notes. \n\nObservaci\u00f3: totes les vostres notes seran enviades a la paperera.
```

Comment:
But your right... i can see now that if i open that file with Geany (another editor) then it shows:

... n\nObservaci\u00f3 ... 

on line#13. SO... i really cannot understand what's happening here!? I checked the charset of the encoding of that file and it is utf-8... so i cannot understand how cant it be wrong encoded!?